docker/dockerfile.tf

```
@@ -25,6 +25,44 @@ COPY --chown=1000:1000 --from=dlfw /usr/local/lib/tensorflow/ /usr/local/lib/ten
 COPY --chown=1000:1000 --from=dlfw /usr/local/lib/python3.8/dist-packages/horovod /usr/local/lib/python3.8/dist-packages/horovod/
 COPY --chown=1000:1000 --from=dlfw /usr/local/bin/horovodrun /usr/local/bin/horovodrun
 
+# Install cmake
+RUN wget http://www.cmake.org/files/v3.21/cmake-3.21.1.tar.gz && \
+    tar xf cmake-3.21.1.tar.gz && cd cmake-3.21.1 && ./configure && make && make install
+
+
+# Install HugeCTR
+ENV LD_LIBRARY_PATH=/usr/local/hugectr/lib:$LD_LIBRARY_PATH \
+    LIBRARY_PATH=/usr/local/hugectr/lib:$LIBRARY_PATH \
+    PYTHONPATH=/usr/local/hugectr/lib:$PYTHONPATH
+
+# Arguments "_XXXX" are only valid when $HUGECTR_DEV_MODE==false
+ARG HUGECTR_DEV_MODE=false
+ARG _HUGECTR_REPO="github.com/NVIDIA-Merlin/HugeCTR.git"
+ARG _CI_JOB_TOKEN=""
+
+RUN mkdir -p /usr/local/nvidia/lib64 && \
+    ln -s /usr/local/cuda/lib64/libcusolver.so /usr/local/nvidia/lib64/libcusolver.so.10
+
+RUN ln -s /usr/lib/x86_64-linux-gnu/libibverbs.so.1 /usr/lib/x86_64-linux-gnu/libibverbs.so
+
+RUN if [ "$HUGECTR_DEV_MODE" == "false" ]; then \
+        git clone https://${_CI_JOB_TOKEN}${_HUGECTR_REPO} build-env && \
+        pushd build-env && \ 
+          git checkout ${HUGECTR_VER} && \
+          cd sparse_operation_kit && \
+          python setup.py install && \
```

If you go `pip install .` this will pick up the build dependencies automatically - and you won't have to install scikit-build/cmake/ninja etc yourself 
```suggestion
          python -m pip install . && \
```